src/foam/mlang/mlang.js

```diff
@@ -1765,8 +1765,12 @@ foam.CLASS({
 
   properties: [
     {
-      class: 'Object',
+      class: 'Double',
```

Comment:
Doesn't this prevent it from working with dates and also convert the type from int/long fields to double?